lib/jekyll/converters/scss.rb

```diff
@@ -92,7 +103,9 @@ def sass_configs
       end
 
       def convert(content)
```

Comment:
I think we would need an override for [`sassify` and `scssify` filters](https://github.com/jekyll/jekyll/blob/05285798be8c43b985f64ebd62854a176b36a0d5/lib/jekyll/filters.rb#L18-L38), which would likely _not_ want this.